libs/utils/utils.js

```diff
@@ -1004,7 +1005,7 @@ async function documentPostSectionLoading(config) {
   delayed([getConfig, getMetadata, loadScript, loadStyle, loadIms]);
 
   import('../martech/attributes.js').then((analytics) => {
-    document.querySelectorAll('main > div').forEach((section, idx) => analytics.decorateSectionAnalytics(section, idx, config));
+    document.querySelectorAll('main > div').forEach((section, idx) => analytics.decorateSectionAnalytics(section, idx, getConfig()));
```

Comment:
Seems like an odd. Change. What does this change get you?

Reply:
@auniverseaway  The config was missing the "analyticLocalization" node and calling getConfig got me the latest and greatest.  But looking at it with fresh eyes I realize that it's nothing in the documentPostSectionLoading function that's adding this data, but actions leading up to the function being called.  So I moved the getConfig call to line 1081 so getConfig is only called once, not in a loop.

Reply:
What is `analyticLocalization` node? I want to make sure we are not putting things in config that don't need to be there.

Reply:
@auniverseaway So as each martech-attribute block is decorated, it adds that key/value pair to a node in config called analyticLocalization.  Then when the sections and block analytics are added and/or a gnav link is getting an analytics value, the object is checked to see if an EN version is available or not.  Reasons I need to store the info versus using it right away:
1.  the information is in a block that will get "decorated" during the block decoration process and timing.  But to not interfere with LHS, the analytics are added in the documentPostSectionLoading function
2. I wanted to avoid requiring the block be decorated before the links it impacts.  If the block can go anywhere in the page (like at the end near the metadata table) it will be less prone to user error.
3. I assumed that there could be more than 1 martech-attribute block (in particular in fragments and in gnav menus).